### DISCUSSION TITLE
Plotman mover

### DISCUSSION THREAD

**Discussion by ghost** [guydavis/machinaris#845] [Ideas]
Hi is there a way to make plotter not plot untill archiver finishes moving?

**Comment by guydavis** [comment]
Good question, unfortunately the original Plotman project does not provide this capability as the `plotman plot` and the `plotman archive` loops run as independent processes, not communicating with each other.

The current workaround is to set the [dst](https://github.com/guydavis/machinaris/wiki/Plotman#destination-storage) directory or directories, to be a fast set of SSD or raided fast HDDs.  This `dst` path is used to locally cache plots created on [tmp](https://github.com/guydavis/machinaris/wiki/Plotman#temporary-storage) before they are [archived](https://github.com/guydavis/machinaris/wiki/Plotman#archiving) (locally or remotely).  Ideally the archiver process should then distribute new plots to multiple target drives concurrently.

That's the theory anyways, when this plotting manager gained prominence for the original Chia plotter at the start of 2021.  Now, with super-fast GPU plotting on the way, this entire pipeline is being called into question.  I will have to look at options.  All feedback is appreciated.

**Comment by ghost** [comment]
Yeah the thing is the archiver takes processor cycles enough to add 20 minutes to plotting. I went from 28mins per plot to  48 because archiver is moving from independant drive to my array in unraid server and main chia node. Oh well. Not really looking for the gpu stuff but if it works as good as machinaris i can give it a shot.
Cjeers mate.
